docs/quickstart.rst

```diff
@@ -53,10 +53,22 @@ Running the Vagrant box
 
 The image is available in https://atlas.hashicorp.com/atomicapp/boxes/dev . However you dont have to download the box image manually. The following steps will take care of it implicitly.
 
+*This Vagrant box is built for VirtualBox. *
+
+To run it on libvirt, a vagrant plugin, `vagrant-mutate <https://github.com/sciurus/vagrant-mutate>`_ must be installed.
+To install vagrant-mutate on *NIX system, run
+
+::
+   $vagrant plugin install vagrant-mutate
+
```

Comment:
@LalatenduMohanty My bad. I didn't look at the URL since I always did a Vagrant init. This means that Just the libvirt plugin has to be installed and nothing more - if it doesn't work.
Thus, I change it to something like, if you are using libvirt and vagrant up gives an error, then do `vagrant plugin install vagrant-libvirt`. Is that fine? Cause I guess that'll do.